### PR TITLE
Prepare 3.2.0 release

### PR DESCRIPTION
**Pull request by autophagy**
@mfussenegger As you can see, this release seems to include fixes. Normally this isn't done, but I suppose is because we are doing this release before 3.1.4. Do you think we should not include the fixes in the release notes for this release, as they will be part of 3.1.4?

### CODE REVIEW

**Comment by autophagy** [comment]
@mfussenegger Have added an upgrades notes section with deprecation and logging changes (https://github.com/crate/crate/pull/8017/commits/931e160115eaf11a8361eadf324bd149f1b2c611#diff-61dad610f55cda2300e84edc0d1f9889R174) similar to how we did it with 3.0 (https://github.com/crate/crate/blob/master/blackbox/docs/appendices/release-notes/3.0.0.rst#upgrade-notes).

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+- Values of byte-size and time based configuration setting do not require a unit
+  suffix anymore. Without a unit time values are treat as milliseconds since
+  epoch and byte size values are treat as bytes.
+
+- Added support of using units inside byte-size or time bases statement
+  parameters values. E.g. '1mb' for 1 MegaByte or '1s' for 1 Second.
+
+.. _version_3.2.0_upgrade_notes:
+
+Upgrade Notes
```
Comment (mfussenegger) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]: I'd put these before the list of changes, but no strong opinion.

File: blackbox/docs/appendices/release-notes/3.2.0.rst [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+- Added support for ``SHOW parameter_name | ALL`` to retrieve one or all session
+  setting value(s).
+
+- Added support for ``INITCAP(string)`` and ``CURRENT_DATABASE`` scalar expressions.
+
+- Improved the compatibility with postgres clients that use the ``text`` type
+  for parameter encoding.
+
+-  Upgraded the Admin UI to 1.11.2 which includes the following changes:
```
Comment (mfussenegger) [inline blackbox/docs/appendices/release-notes/3.2.0.rst]: I think this is now `1.11.3` ?

File: blackbox/docs/appendices/release-notes/3.2.0.rst (hunk 1 above)
Comment (autophagy) [reply blackbox/docs/appendices/release-notes/3.2.0.rst]: Yeah, I think I'd agree.

File: blackbox/docs/appendices/release-notes/3.2.0.rst (hunk 2 above)
Comment (autophagy) [reply blackbox/docs/appendices/release-notes/3.2.0.rst]: Good catch!

**Pull request merged**